packages/worker/src/entities/addressTransaction.entity.ts

```diff
@@ -35,4 +35,7 @@ export class AddressTransaction extends BaseEntity {
 
   @Column({ type: "int" })
   public readonly transactionIndex: number;
+
+  @Column({ type: "boolean", nullable: true })
+  public readonly isEvmLike?: boolean;
```

Comment:
I don't think this field should be here, as this entity is used to query transactions based on an address and join the corresponding transaction entities, which have all the fields available: https://github.com/matter-labs/block-explorer/blob/0b1a667f13fb4397c2fe78e401a77d4a0842e57c/packages/api/src/transaction/transaction.service.ts#L62